README.md

```diff
@@ -1,3 +1,26 @@
-# CP1404 Practicals
+# CP1404 practicals
+
+## Practical work for IT@JCU CP1404
+
+## Brief information about the repo and yourself.
+
+
+My name is Shwe Yamin Oo ,and I am studying data science major in James Cook University, Singapore.Among the subjects that i took in this trimester, CP1404 is a subject that I am most interested.
```

Comment:
Nice and clear formatting, includes relevant info

Reply:
Ok i will add that .
Thanks you